src/MIDebugEngine/AD7.Impl/AD7Disassembly.cs

```diff
@@ -162,6 +176,28 @@ public int Read(uint dwInstructions, enum_DISASSEMBLY_STREAM_FIELDS dwFields, ou
                     }
                 }
 
+                if (!string.IsNullOrWhiteSpace(instruction.File))
```

Comment:
>if [](http://example.com/codeflow?start=16&length=2)

We need to compute `DF_DOCUMENTCHANGE`. Note that that flag isn't trivial as it requires either going back an extra instruction to compute, or remembering where we left off in the case of multiple calls to `Next` without a call to Seek.